app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/utils/Constant.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.marcinwisniewski.intivetrainingapp.utils
+
+class Constant {
+    companion object {
```

Comment:
If those statics are not shared we should avoid java style.
More here: https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/

Reply:
Changed in first commit after review.